src/smefit/optimize/ultranest.py

```diff
@@ -128,39 +127,61 @@ def from_dict(cls, config):
+                )
+                ns_settings["ReactiveNS_settings"]["resume"] = "overwrite"
+
+        ns_settings["sampling_settings"] = ns_settings.get("sampling_settings", {})
+
+        if "min_num_live_points" not in ns_settings["sampling_settings"]:
+            min_num_live_points_def = 500
             _logger.warning(
-                f"Number of live points (nlive) not set in the input card. Using default: {nlive}"
+                f"Number of live points (nlive) not set in the input card. Using default: {min_num_live_points_def}"
             )
+            ns_settings["sampling_settings"]["min_num_live_points"] = min_num_live_points_def
 
-        lepsilon = config.get("lepsilon", 0.05)
-        if "lepsilon" not in config:
+        if "Lepsilon" not in ns_settings["sampling_settings"]:
+            Lepsilon_def = 0.05
             _logger.warning(
-                f"Sampling tollerance (Lepsilon) not set in the input card. Using default: {lepsilon}"
+                f"Sampling tollerance (Lepsilon) not set in the input card. Using default: {Lepsilon_def}"
             )
+            ns_settings["sampling_settings"]["Lepsilon"] = Lepsilon_def
 
-        target_evidence_unc = config.get("target_evidence_unc", 0.5)
-        if "target_evidence_unc" not in config:
+        if "dlogz" not in ns_settings["sampling_settings"]:
+            dlogz_def = 0.5
             _logger.warning(
-                f"Target Evidence uncertanty (target_evidence_unc) not set in the input card. Using default: {target_evidence_unc}"
+                f"Target Evidence uncertanty (dlogz) not set in the input card. Using default: {dlogz_def}"
             )
+            ns_settings["sampling_settings"]["dlogz"] = dlogz_def
 
-        target_post_unc = config.get("target_post_unc", 0.5)
-        if "target_post_unc" not in config:
+        if "dKL" not in ns_settings["sampling_settings"]:
+            dKL_def = 0.5
             _logger.warning(
-                f"Target Posterior uncertanty (target_post_unc) not set in the input card. Using default: {target_post_unc}"
+                f"Target Posterior uncertanty (dKL) not set in the input card. Using default: {dKL_def}"
             )
+            ns_settings["sampling_settings"]["dKL"] = dKL_def
 
-        frac_remain = config.get("frac_remain", 0.01)
-        if "frac_remain" not in config:
+        if "frac_remain" not in ns_settings["sampling_settings"]:
+            frac_remain_def = 0.01
             _logger.warning(
-                f"Remaining fraction (frac_remain) not set in the input card. Using default: {frac_remain}"
+                f"Remaining fraction (frac_remain) not set in the input card. Using default: {frac_remain_def}"
             )
+            ns_settings["sampling_settings"]["frac_remain"] = frac_remain_def
 
-        store_raw = config.get("store_raw", False)
+        #store_raw = ns_settings.get("store_raw", False)
```

Comment:
If I recall correctly storing the intermediate results was quite heavy in memeory.
Is there still the possibility of not storing the partial results?

`store_raw` is used also below to determine the `log_dir`